pkg/cmd/application/create/app_create.go

```diff
@@ -121,11 +129,19 @@ NOTE:
 }
 
 func run(opts *AppCreateOpt, cfg *config.Config) error {
-	if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.CPlatform); err != nil {
-		return err
+	defer opts.Streams.CurrentHandler.Done()
+
+	opts.Streams.CurrentHandler.Info(fmt.Sprintf("resetting local repository [repo=%s]", cfg.Repositories.CPlatform.Value))
+	if !cfg.DryRun {
+		if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.CPlatform, cfg.DryRun); err != nil {
+			return err
+		}
 	}
-	if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.Templates); err != nil {
-		return err
+	opts.Streams.CurrentHandler.Info(fmt.Sprintf("resetting local repository [repo=%s]", cfg.Repositories.Templates.Value))
+	if !cfg.DryRun {
+		if _, err := config.ResetConfigRepositoryState(&cfg.Repositories.Templates, cfg.DryRun); err != nil {
+			return err
```

Comment:
same here :) 

something like 
return fmt.Errorf("failed to reset config repository state for Templates: %w", err)

Reply:
Added